### PR TITLE
Make the first homework (#9 variant)

### PR DESCRIPTION
**Pull request by Sontsevolosa**

### CODE REVIEW

**Review by oleksandr-kychun — COMMENTED** [review]
i like your solution) have you thought about removing duplication?)

File: homework.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
-VARIANT = 0
+VARIANT = 9
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  first_random_numbers = Array.new(3) { rand(1...9) }.join('')
+  second_random_numbers = Array.new(3) { rand(1...9) }.join('')
```
Comment (sveredyuk) [inline homework.rb]: DRY

File: homework.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+  letters_array = ('A'..'Z').to_a.flatten
+  first_random_letters = Array.new(3) { letters_array[rand(letters_array.length)] }.join('')
+  second_random_letters = Array.new(3) { letters_array[rand(letters_array.length)] }.join('')
```
Comment (sveredyuk) [inline homework.rb]: DRY

File: homework.rb [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+
+  "#{first_random_numbers}" + "-" + "#{first_random_letters}" + "-" + "#{second_random_letters}" + "-" + "#{second_random_numbers}"
```
Comment (sveredyuk) [inline homework.rb]: DRY

**Comment by Sontsevolosa** [comment]
@ghostinshadow @sveredyuk please, review my code)

**Comment by Sontsevolosa** [comment]
@ghostinshadow @sveredyuk please, review my code)

File: homework.rb [hunk 4]
```diff
@@ -1,6 +1,10 @@
...
+VARIANT = 9
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  random_numbers = Array.new(3) { rand(1...9) }.join('')
+
+  letters_array = ('A'..'Z').to_a.flatten
+  random_letters = Array.new(3) { letters_array[rand(letters_array.length)] }.join('')
+
```
Comment (sveredyuk) [inline homework.rb]:
```ruby
def random(things)
  if things == :numbers
    Array.new(3) { rand(1...9) }.join('')
  elsif things == :letters
    Array.new(3) { letters_array[rand(letters_array.length)] }.join('')
  else
    raise 'Unknown thing'
  end
end
```

File: homework.rb [hunk 5]
```diff
@@ -1,6 +1,10 @@
...
+  "#{random_numbers}-#{random_letters}-#{random_letters}-#{random_numbers}"
```
Comment (sveredyuk) [inline homework.rb]: "#{random(:numbers)}-#{random(:letters)}-#{random(:letters)}-#{random(:numbers)}"

File: homework.rb [hunk 6]
```diff
@@ -1,6 +1,10 @@
-VARIANT = 0
+VARIANT = 9
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  random_numbers = Array.new(3) { rand(1...9) }.join('')
```
Comment (sveredyuk) [inline homework.rb]: redundant

File: homework.rb [hunk 7]
```diff
@@ -1,6 +1,10 @@
...
+
+  letters_array = ('A'..'Z').to_a.flatten
+  random_letters = Array.new(3) { letters_array[rand(letters_array.length)] }.join('')
```
Comment (sveredyuk) [inline homework.rb]: redundant

**Comment by Sontsevolosa** [comment]
@sveredyuk please, review my changes)

**Review by sveredyuk — APPROVED** [review]

**Pull request closed without merging**